### PR TITLE
refactor(dedicated.cdn): remove responsive-tabs

### PR DESCRIPTION
**Pull request by FredericEspiau**
MANAGER-4332

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.html [hunk 1]
```diff
@@ -1,14 +1,12 @@
-<div class="cdn-dedicated">
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.html]: Removed unused `div`

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html [hunk 2]
```diff
@@ -1,54 +1,43 @@
-<div class="cdn-dedicated-manage">
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html]: Removed unused `div`

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html [hunk 3]
```diff
@@ -8,11 +8,7 @@
             'text-danger'  : log.message.status > 499
         }"
     >
-        <span
-            data-ng-bind="log.message.source"
-            title="ip_source"
-            class="col-md-1"
-        ></span>
+        <span data-ng-bind="log.message.source" title="ip_source"></span>
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html]: Fix tiny display bug

File: packages/manager/apps/dedicated/client/app/common/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -134,6 +134,7 @@
   "common_service_expired": "Le service {{t0}} n'a pas été renouvelé. Cela est peut être dû à une décision de votre part ou un retard de paiement de votre côté.",
   "common_reset_zoom": "Réinitialiser le zoom",
   "common_alerts_message_see_more": "Voir le détail des erreurs",
+  "common_alerts_message_see_less": "Masquer le détail des erreurs",
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/common/translations/Messages_fr_FR.json]: Fix missing translation

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html [hunk 5]
```diff
@@ -1,59 +1,54 @@
...
-    <!-- End of Tabs -->
-
-    <!-- Tab Content -->
-    <div class="tab-content">
-        <div
-            class="alert alert-dismissible"
-            role="alert"
-            data-ng-class="alertType"
-            data-ng-if="message"
+<!-- Tab Content -->
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html]: No changes, you are looking at whitespaces

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html [hunk 6]
```diff
@@ -1,54 +1,43 @@
...
-            <div
-                class="alert"
-                role="alert"
-                data-ng-class="alertType"
-                data-ng-if="message"
-            >
-                <i class="close icon-remove" data-ng-click="message = null"></i>
-                <span data-ng-bind="message"></span>
-            </div>
+    <!-- Tab Content -->
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html]: No changes, you are looking at whitespaces

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html [hunk 7]
```diff
@@ -1,140 +1,138 @@
-<div data-ng-controller="CdnTabSslCtrl">
-    <div class="text-center" data-ng-show="loading">
-        <oui-spinner></oui-spinner>
-    </div>
+<div class="text-center" data-ng-show="loading">
```
Comment (FredericEspiau) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: No changes, you are looking at whitespaces

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+angular.module('App').config(($stateProvider) => {
+  $stateProvider.state('app.networks.cdn.dedicated.manage', {
+    redirectTo: 'app.networks.cdn.dedicated.manage.statistics',
+    translations: { value: ['.'], format: 'json' },
+    views: {
+      cdnMainView: {
+        templateUrl: 'cdn/dedicated/manage/cdn-dedicated-manage.html',
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js]: Could be imported then (tell webpack to import it)?

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html (hunk 3 above)
Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html]: Only columns (`.col-*-*`) may be children of `.rows`

See: https://github.com/twbs/bootlint/wiki/E013

But I'm wondering why a `ml-0` is used on the parent element.

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js [hunk 9]
```diff
@@ -0,0 +1,13 @@
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */ ($stateProvider) => {
```

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js [hunk 10]
```diff
@@ -0,0 +1,12 @@
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */ ($stateProvider) => {
```

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js [hunk 11]
```diff
@@ -0,0 +1,12 @@
...
+  $stateProvider.state('app.networks.cdn.dedicated.manage.statistics', {
+    views: {
+      cdnView: {
+        templateUrl:
+          'cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.html',
```
Comment (antleblanc) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js]: Could be imported.

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html (hunk 3 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html]: Nice catch, I'll remove the row altogether

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js (hunk 8 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js]: Could be, but it's not the point of this PR 🙂

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html (hunk 3 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html]: I've removed both, the display is the same ¯\_(ツ)_/¯

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html [hunk 12]
```diff
@@ -1,59 +1,54 @@
...
+        class="alert alert-dismissible"
+        role="alert"
+        data-ng-class="alertType"
+        data-ng-if="message"
+    >
+        <button
+            type="button"
+            class="close"
+            data-dismiss="alert"
+            aria-label="Close"
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html]: Done, but now we need translations 🙂

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js (hunk 8 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js]: Done

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js (hunk 9 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.routes.js]: Done

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js (hunk 10 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js]: Done

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js (hunk 11 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/cdn-dedicated-manage-statistics.routes.js]: Done

File: packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js [hunk 13]
```diff
@@ -12,9 +12,7 @@ angular
     'HostingDatabaseCreateCtrl',
     (
       $location,
-      $q,
```
Comment (FredericEspiau) [inline packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js]: Those were unsused

**Comment by FredericEspiau** [comment]
@antleblanc I've added a new translations now because of the `Close` tag you asked me to remove, so now we need translations 🙂

**Comment by ovh-cds** [comment]
CDS Report installation#8880.0 ✘  
* Stage 1
  * Install ✘

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.service.js [hunk 14]
```diff
@@ -137,7 +137,8 @@ angular
           }
           return $q.reject(cdn);
         })
-        .then(() => result, (http) => $q.reject(http.data));
+        .then(() => result)
+        .catch((http) => $q.reject(http.data));
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.service.js]: This doesn't seem related to remove `responsive-tabs`, if it's this type of change is only made once, it's ok but if it's done a few times myabe it should be brought by its own commit

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html [hunk 15]
```diff
@@ -1,59 +1,54 @@
...
+                }"
             >
             </span>
         </h1>
     </div>
-    <!-- End of Header -->
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            text="{{:: 'cdn_domain_tab_statistics' | translate }}"
+            state="app.networks.cdn.dedicated.domain.statistics"
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html]: Could have been nice to extract those through the router's resolve if possible

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/statistics/translations/Messages_fr_FR.json [hunk 16]
```diff
@@ -0,0 +1,32 @@
+{
+  "cnd_stats_filters_visualize": "Visualiser",
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/statistics/translations/Messages_fr_FR.json]: This looks more like splitting translations than removing tabs

File: packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js (hunk 13 above)
Comment (marie-j) [reply packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js]: Nice catch, but it should be done in a different commit given the scope

**Comment by FredericEspiau** [comment]
I've splitted into a lot of commits because @marie-j was right about this subject (as usual)

File: packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js (hunk 13 above)
Comment (FredericEspiau) [reply packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js]: Done

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html (hunk 15 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html]: Yes, could have 🌞

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.service.js (hunk 14 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/cdn-dedicated.service.js]: Done !

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html [hunk 17]
```diff
@@ -1,140 +1,138 @@
...
+                    <dl class="oui-tile__definition">
+                        <dt
+                            class="oui-tile__term"
+                            data-translate="cdn_configuration_ssl_status"
+                        ></dt>
+                        <dd class="oui-tile__description">
+                            <span
+                                class="oui-status"
+                                data-ng-class="getBadgeClass(ssl.status)"
+                                data-ng-bind="'cdn_configuration_ssl_state_' + ssl.status.toUpperCase() | translate"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: one time binding may be?

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html [hunk 18]
```diff
@@ -1,59 +1,54 @@
...
-                aria-label="Close"
-                data-ng-click="message = null"
-            >
-                <span aria-hidden="true">&times;</span>
-            </button>
-            <span data-ng-bind="message"></span>
-        </div>
-
-        <div data-ui-view="cdnDomainView"></div>
+            <span aria-hidden="true">&times;</span>
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html]: Thanks for your input 🙂 

You are looking at whitespace changes, I don't write this part of the code !

you can remove whitespace changes by clicking on the gear icon and on *Hide whitespace changes*

I won't change anything that is in the whitespace as it's not part of this PR 😄

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html [hunk 19]
```diff
@@ -1,59 +1,54 @@
...
+                    'oui-status_error': $ctrl.domain.mode ==='OFF'
+                }"
             >
             </span>
         </h1>
     </div>
-    <!-- End of Header -->
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            text="{{:: 'cdn_domain_tab_statistics' | translate }}"
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/domain/cdn-dedicated-domain.html]: We don't need them for the components ! This is why I didn't add them !

You still need them on a `button` though !

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html [hunk 20]
```diff
@@ -1,54 +1,43 @@
...
+    <div class="tab-content">
+        <div
+            class="alert"
+            role="alert"
+            data-ng-class="alertType"
+            data-ng-if="message"
+        >
+            <i class="close icon-remove" data-ng-click="message = null"></i>
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html]: Whitespaces 😄

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html [hunk 21]
```diff
@@ -1,140 +1,138 @@
...
-            >
-            </a>
-            <a
-                class="btn btn-block btn-default"
-                data-ui-sref=".generate"
-                data-translate="cdn_configuration_generate_ssl_title"
-            >
-            </a>
-        </div>
+<div class="row d-md-flex mb-5" data-ng-show="ssl == null && !loading">
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: Whitespaces 😄

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html (hunk 17 above)
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: Whitespaces 😄

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html [hunk 22]
```diff
@@ -1,140 +1,138 @@
...
-                                class="oui-tile__term"
-                                data-translate="cdn_configuration_ssl_certificateProvider"
-                            ></dt>
-                            <dd
-                                class="oui-tile__description"
-                                data-translate="{{ 'cdn_configuration_ssl_provider_' + ssl.certificateProvider }}"
-                            ></dd>
-                        </dl>
-                    </div>
+<div class="row d-md-flex mb-5" data-ng-show="ssl != null && !loading">
```
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: Whitespaces 😄

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html (hunk 21 above)
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/ssl/cdn-dedicated-manage-ssl.html]: Whitespaces 😄

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html (hunk 20 above)
[the remark this answers is not in the document]
Comment (FredericEspiau) [reply packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html]: Whitespaces 😄

**Comment by FredericEspiau** [comment]
@radireddy thanks for your valuable input 🙏 

About your remarks:
- you made some comments about changes in the whitespaces and I won't fix them as it's not part of this PR, even if the remarks are correct !
- I removed the `data-` because I was on Component and it is valid html5 ! I should have talked to you in India though, because we don't need them in this case

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/translations/Messages_fr_FR.json [hunk 23]
```diff
@@ -0,0 +1,143 @@
...
+  "cdn_configuration_cacherules_upgrade_step1_tarif2": "{{t0}} HT / mois pour",
+  "cdn_configuration_cacherules_upgrade_step1_tarif2_rules_count": "{{t0}} règles de cache",
+  "cdn_configuration_cacherules_upgrade_step2_durations": "Commander {{t0}} règles de cache supplémentaires pendant :",
+  "cdn_configuration_cacherules_upgrade_step2_duration_total": "{{t0}} mois : total {{t1}} HT",
+  "cdn_configuration_cacherules_upgrade_step3_question": "Ajout de {{t0}} règles de cache supplémentaires sur le CDN <span class=\"italic normal\">{{t1}}</span>",
+  "cdn_configuration_cacherules_upgrade_step3_id": "Afin de confirmer votre demande, veuillez accepter les conditions d'utilisation et cliquer sur le bouton \"Suivant\" ci-dessous.",
+  "cdn_configuration_cacherules_upgrade_step3_totalHT": "Prix HT :",
+  "cdn_configuration_cacherules_upgrade_step3_totalTTC": "Prix TTC :",
+  "cdn_configuration_cacherules_upgrade_step3_contracts": "Je reconnais avoir bien pris connaissance",
+  "cdn_configuration_cacherules_upgrade_step3_contracts_of": "des",
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/translations/Messages_fr_FR.json]: i don't think it'll be correctly translated

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/translations/Messages_fr_FR.json [hunk 24]
```diff
@@ -0,0 +1,143 @@
...
+  "cdn_configuration_cacherules_upgrade_step4_bc": "Le bon de commande a été généré avec succès.",
+  "cdn_configuration_cacherules_upgrade_step4_continue": "Afin de confirmer votre commande, veuillez cliquer sur le bouton \"Régler\" ci-dessous.",
+  "cdn_configuration_cacherules_upgrade_step4_explication": "Veuillez noter qu'un délai d'environ 2 heures à compter de l'acceptation de votre paiement est nécessaire pour que l'activation de l'option soit prise en compte dans votre Espace Client Dédié.",
+  "cdn_configuration_cacherules_upgrade_step4_explication2": "Un nouvel onglet s'ouvrira et vous serez automatiquement redirigé(e) vers l'interface de paiement OVH.",
+  "cdn_configuration_cacherules_upgrade_fail": "Une erreur est survenue lors du chargement des données",
+  "cdn_configuration_cacherules_upgrade_subscribed": "Votre service est déjà configuré avec cette option",
+
+  "cnd_stats_filters_visualize": "Visualiser",
+  "cnd_stats_filters_from": "de",
+  "cnd_stats_filters_to": "sur",
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/statistics/translations/Messages_fr_FR.json]: same here how the translator are supposed to handle that ?

File: packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html (hunk 20 above)
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/cdn-dedicated-manage.html]: i'm not sure ```<i  ng-click>``` is accessible
